### PR TITLE
Exclude VMR package cache from assembly scanning

### PR DESCRIPTION
**Pull request by mmitche**
When building in the VMR, the per-repo package cache is placed under the repo's artifacts directory. Exclude this when doing the assembly scan.

In addition, fix up the file to be compatible with non-windows systems.

### CODE REVIEW

**Comment by mmitche** [comment]
Apologies if this is completely wrong or needs serious refactoring. I know about as much F# as I know Dutch. Which is none.

**Review by KevinRansom — APPROVED** [review]

**Comment by mmitche** [comment]
@KevinRansom Can I get another review on this to unblock the VMR work?

**Review by vzarytovskii — APPROVED** [review]

**Pull request merged**
